packages/manager/apps/hycu/public/translations/listing/Messages_fr_FR.json

```diff
@@ -1,5 +1,5 @@
 {
-  "title": "HYCU",
+  "description": "Simplifiez vos sauvegardes, plans de reprise après sinistre et migrations de votre infrastructure Nutanix. Ce service vous propose différents packs de licences HYCU Hybrid Cloud pour protéger vos charges de travail Nutanix.",
```

Comment:
Should be better if all keys follow the same naming rule -> `hycu_` then.
Moreover, this text also appears in the onboarding part so you could move it in a parent resource file to declare it only once